x-pack/plugins/ml/server/lib/check_annotations_index/index.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { callWithInternalUserFactory } from '../../client/call_with_internal_user_factory';
+
+import {
+  ML_ANNOTATIONS_INDEX_ALIAS_READ,
+  ML_ANNOTATIONS_INDEX_ALIAS_WRITE,
+  ML_ANNOTATIONS_INDEX_PATTERN
+} from '../../../common/constants/index_patterns';
+
+export async function checkAnnotationsIndex(server) {
```

Comment:
Once the backend code is written this function should just do checks.  It shouldn't create or delete any indices or aliases.  For the upgrade from 7.x to 8.x the backend will need to switch the aliases, and we don't want the UI to work against this.

Reply:
Fixed in 4d65b16.